melodic/distribution.yaml

```diff
@@ -6235,6 +6235,22 @@ repositories:
       type: git
       url: https://github.com/NiryoRobotics/niryo_one_ros_simulation.git
       version: master
+  nmc_nlp_lite-release:
+    doc:
+      type: git
+      url: https://github.com/nmcbins/nmc_nlp_lite_ros.git
+      version: master
+    release:
+      packages:
+      - nmc_nlp_lite
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/nmcbins/nmc_nlp_lite-release.git
```

Comment:
While it is not required to do so, most other packages use the name of the source repository with `-release` appended.  In this case, that would be:

```
https://github.com/nmcbins/nmc_nlp_lite_ros-release.git
```

Reply:
Thanks a lot!  Will fix and redo!

Reply:
Great advice nonetheless -- modified as suggested. Thanks!